migration/migrator/data/submitty_db.sql

```diff
@@ -220,7 +220,7 @@ ALTER TABLE ONLY courses_users
 
 
 ALTER TABLE ONLY emails
-    ADD CONSTRAINT emails_user_id_fk FOREIGN KEY (user_id) REFERENCES users(user_id) ON UPDATE CASCADE;
+    ADD CONSTRAINT emails_user_id_fk FOREIGN KEY (user_id) REFERENCES users(user_id) ON UPDATE CASCADE ON DELETE CASCADE;
```

Comment:
We should add a similar `ON DELETE CASCADE` to `sessions` foreign key constraint